### ISSUE TITLE
'string' is no longer a member of 'google::protobuf'

### ISSUE THREAD

**Issue by tusharb86** [protocolbuffers/protobuf#9943]
**What version of protobuf and what language are you using?**
Version: v3.20.0
Language: C++

**What operating system (Linux, Windows, ...) and version?**
Windows 11

**What runtime / compiler are you using (e.g., python version or gcc version)**
Visual Studio - MSVC 14.29

**What did you do?**
I updated from protobuf version 3.19.4 to 3.20.1 for my project and started seeing a build error:
`error C2039: 'string': is not a member of 'google::protobuf'`

**What did you expect to see**
I expected to see no build errors since this is just a minor version change.

**What did you see instead?**
I'm seeing build errors like this:
`error C2039: 'string': is not a member of 'google::protobuf'`

**Details**
I compared the headers between version 3.19.4 and 3.20.0 and noticed an important difference in the file src/google/protobuf/stubs/common.h. Specifically, the following lines were removed:
```c++
// This is at the end of the file instead of the beginning to work around a bug
// in some versions of MSVC.
using std::string;
```

This means that code can no longer use `google::protobuf::string`. This seems like an unintended breaking change. @acozzette - This seems to have happened between [Jan 27](https://github.com/protocolbuffers/protobuf/blob/22d0e265de7d2b3d2e9a00d071313502e7d4cccf/src/google/protobuf/stubs/common.h) and [Jan 28](https://github.com/protocolbuffers/protobuf/blob/18521f60556b48be8beb6623dc6cb0c0ed80a79b/src/google/protobuf/stubs/common.h) for the `common.h` header file. Was the intention to move the `using` statement to the beginning of the file?

Thanks for looking into this issue!

**Comment by acozzette** [comment]
Thank you for the bug report, but this was actually an intentional change. We used to have `using namespace std;` in this file to allow us to refer to anything in `std::` with a shorter name, but we cleaned that up a while ago enough so that the only symbol we referred to in this way was `std::string`. And then more recently we finally cleaned that up as well, and deleted the `using std::string` line. Is it possible to update your code to refer to `std::string` instead of `google::protobuf::string`?
